.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  # Enable version updates for Docker
+  - package-ecosystem: "docker"
+    # Look for a `Dockerfile` in the `build` directory
+    directory: "/build"
+    # Check for updates once a week
+    schedule:
+      interval: "weekly"
```

Comment:
```suggestion
      interval: "weekly"

```

Reply:
and yeah, maybe let's start with daily and if it will spam us too much, we'll be able to return back to weekly